testtools/compat.py

```diff
@@ -86,6 +85,7 @@ def classtypes():
         return (type, types.ClassType)
     str_is_unicode = sys.platform == "cli"
     text_or_bytes = (unicode, str)
+    text = unicode
```

Comment:
I think we have a dependency on six now, so lets just use six.text_type rather than adding to this module?